sirius-valley/lerni-mobile
# [LER-219] Html support on bubbles

On this PR: 
- html support for bubbles. 

To test this feature: 
- Open the devtools in the network tab.
- Go to a pill, se the responses with html code and watch the bubbles. No html tag should be render.
